compiler/src/iree/compiler/GlobalOptimization/test/materialize_homogeneous_encodings.mlir

```diff
@@ -96,3 +96,70 @@ module attributes {hal.device.targets = [#device_target_vulkan, #device_target_l
 // CHECK-LABEL: func.func @lhs_encoding
 // CHECK-SAME:    %[[ARG0:[a-zA-Z0-9]+]]
 // CHECK:         return %[[ARG0]]
+
+// -----
+
+#executable_target_rocm_hsaco_fb = #hal.executable.target<"rocm", "rocm-hsaco-fb", {target_arch = "gfx90a", ukernels = "none"}>
```

Comment:
this is a lot of IR for something that doesn't need it all - can you reduce this?
(every additional line of IR we have in lit tests that aren't relevant to the test becomes a burden if anything changes)